webpack.config.js

```diff
@@ -79,10 +70,6 @@ module.exports = {
   optimization: {
     minimize: true,
     minimizer: [
-      new TerserPlugin({
-        test: /\.js(\?.*)?$/i,
-        extractComments: false,
```

Comment:
HtmlBundlerPlugin per default avoid the extraction of comments from source files